modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 6.4 - 2023-03-15
+
```

Comment:
```suggestion
## 6.4 - 2023-03-15

```

This will be added in the release process

Reply:
Removed.